### PR TITLE
fix(studio): fix error when renaming node

### PR DESCRIPTION
**Pull request by sebburon**
fixes https://github.com/botpress/studio/issues/52

Added a toast at the bottom of the screen.
Screen Shot 2021-11-01 at 11 29 18 AM

### CODE REVIEW

**Comment by allardy** [comment]
I'm getting a weird bug. Create a new flow, then add a new node on that flow.
Click on the node to open the inspector. Then click outside the node (without changing anything), and you will get the error toast. Add other nodes, it will repeat the same issue. Switching a node and clicking on the background triggers the toast

**Comment by sebburon** [comment]
> I'm getting a weird bug. Create a new flow, then add a new node on that flow.
> Click on the node to open the inspector. Then click outside the node (without changing anything), and you will get the error toast. Add other nodes, it will repeat the same issue. Switching a node and clicking on the background triggers the toast

The node is being renamed every time we inspect it...
Working on it !

File: packages/studio-ui/src/web/views/FlowBuilder/nodeProps/StandardNode.jsx [hunk 1]
```diff
@@ -12,9 +12,11 @@ const style = require('./style.scss')
 
 export default class StandardNodePropertiesPanel extends Component {
   renameNode = text => {
-    if (text) {
+    if (text !== this.props.node.name) {
```
Comment (allardy) [inline packages/studio-ui/src/web/views/FlowBuilder/nodeProps/StandardNode.jsx:15]: This brings another issue. An empty name will create a node where all the other nodes will point to
image

Also, please rename the method, "renameNode" is not accurate, it's "onChange". So we need to check if text is set and is not blank

Comment (sebburon) [reply packages/studio-ui/src/web/views/FlowBuilder/nodeProps/StandardNode.jsx]: You are right my bad !

**Review by allardy — APPROVED** [review]
LGTM

**Pull request merged**